flink/Dockerfile

```
@@ -1,21 +1,67 @@
+FROM openjdk:11.0.16-slim-buster
```

Since we need only of JRE in runtime, we can save some megas of download here
```suggestion
FROM openjdk:11.0.16-jre-slim-bullseye
```

---

Oh, my mistake, it should've been JRE-only from the start.